pages/package/[...packageString]/components/ExportAnalysisSection/ExportAnalysisSection.tsx

```diff
@@ -92,7 +106,7 @@ function ExportList({ exports, totalSize, isLoading }) {
                   size={exportDictionary[letter][0].gzip}
                   totalSize={totalSize}
                   name={exportDictionary[letter][0].name}
-                  path={exportDictionary[letter][0].path}
+                  // path={exportDictionary[letter][0].path}
```

Comment:
Why did we need to comment this out?

Reply:
> Why did we need to comment this out?

I think the child component does not use `path`